### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Review by aayushjain-code — COMMENTED** [review]
Try to make the cards center allign

**Review by aayushjain-code — APPROVED** [review]

**Comment by MuskanChhatrasal** [comment]
Okayy!! Will do that.

File: Pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star-half-alt"></i>
+                </div>
+                <a href="#" class="btn">Buy Now</a>
+            </div>
+    
+            <div class="box">
+                <img src="../image/s-img-2.jpg" alt="Pizza">
+                <h3>Pizza</h3>
+                <div class="price"> $4.99/- -10.99/-</div>
```
Comment (ApurvaSawant11) [inline Pages/wishlist.html:69]: For all the cards, add `$` before 10.99. And remove `/-` after the price

File: Pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            <div class="fas fa-user" id="login-btn"><a href="./Pages/login.html"></a></div>
+        </div>
+
+    </header>
+
+    <section class="categories" id="categories">
+    
+        <h1 class="wishlist-title">Wishlist</h1>
+    
+        <div class="box-container">
```
Comment (ApurvaSawant11) [inline Pages/wishlist.html:50]: `center` this container and also reduce `padding` of the `categories` section above

File: Pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                    <a href="#" class="fab fa-facebook-f"></a>
+                    <a href="#" class="fab fa-twitter"></a>
+                    <a href="#" class="fab fa-instagram"></a>
+                    <a href="#" class="fab fa-linkedin"></a>
+                </div>
+            </div>
+    
+            <div class="box">
+                <h3>contact info</h3>
+                <a href="#" class="links"> <i class="fas fa-phone"></i> +123-456-7890 </a>
```
Comment (ApurvaSawant11) [inline Pages/wishlist.html:131]: Flip the phone icon. You can add class `fa-rotate-90` or `fa-flip-horizontal` to the icon

File: Pages/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+            <a href="./cart.html">Cart</a>
+            <a href="./login.html">Login</a>
+        </nav>
+
+        <div class="icons">
+            <div class="fas fa-bars" id="menu-btn"></div>
+            <div class="fas fa-shopping-cart" id="cart-btn"><a href="./cart.html"></a></div>
+            <div class="fas fa-user" id="login-btn"><a href="./Pages/login.html"></a></div>
+        </div>
+
```
Comment (ApurvaSawant11) [inline Pages/wishlist.html:43]: You have both, link and icon for `cart`. Is there any specific reason for this? If not, then remove either of them.

**Review by ApurvaSawant11 — APPROVED** [review]
The code is clean and the hover effects are awesome!

Just a few UI suggestions. Rest everything looks great.

File: Pages/wishlist.html (hunk 1 above)
Comment (MuskanChhatrasal) [reply Pages/wishlist.html:69]: Okayy!! Will do that.
Thankyou

File: Pages/wishlist.html (hunk 4 above)
Comment (MuskanChhatrasal) [reply Pages/wishlist.html:43]: I have to remove one. I will do that.

File: Pages/wishlist.html (hunk 3 above)
Comment (MuskanChhatrasal) [reply Pages/wishlist.html:131]: Okayy!!

File: Pages/wishlist.html (hunk 2 above)
Comment (MuskanChhatrasal) [reply Pages/wishlist.html:50]: Okayy!!

File: Pages/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- font awesome cdn link  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+
+    <link rel="stylesheet" href="https://wow-ui02.netlify.app/wowi.css">
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```
Comment (harshkotkar5557) [inline Pages/wishlist.html:15]: you can use your own component library instead of anyother

**Review by harshkotkar5557 — COMMENTED** [review]
clean code

**Pull request merged**